code/game/atoms.dm

```diff
@@ -76,8 +79,8 @@ directive is properly returned.
 /atom/proc/on_reagent_change()
 	return
 
-/atom/proc/Bumped(AM as mob|obj)
-	return
+/atom/proc/Bumped(atom/movable/AM)
+	set waitfor = FALSE
```

Comment:
While this is the /tg/ standard, I'm not entirely sure that's the best for us, as our bump code is not the same as theirs, nor theirs is necessarily the best here.

Reply:
So what should I do

Reply:
Leave this one doing nothing, just returning.